build.gradle

```diff
@@ -1,80 +1,71 @@
-import io.github.httpbuilderng.http.HttpTask
-import groovyx.net.http.HttpBuilder
+import org.asciidoctor.gradle.jvm.AsciidoctorTask
+import com.neo4j.gradle.wordpress.WordPressPlugin
+import com.neo4j.gradle.wordpress.WordPressUploadTask
 
 plugins {
-	id "io.github.http-builder-ng.http-plugin" version "0.1.1"
-    id "org.asciidoctor.convert" version "1.5.9.2"
+    id 'org.asciidoctor.jvm.gems' version '3.1.0' apply false
+    id 'org.asciidoctor.jvm.convert' version '3.1.0' apply false
+    // FIXME: use the published version once parent page will be supported
+    //id "com.neo4j.gradle.wordpress.WordPressPlugin" version "0.0.1"
 }
-asciidoctorj {
-    version = '1.6.1'
+
+apply plugin: 'org.asciidoctor.jvm.gems'
+apply plugin: WordPressPlugin
+
+repositories {
+    mavenCentral()
+    jcenter()
+    ruby {
+        gems()
+    }
 }
 
+wordpress {
+    username = project.hasProperty('wordpress-username') ? project.property('wordpress-username') : 'neo'
+    password = project.hasProperty('wordpress-password') ? project.property('wordpress-password') : ''
+    host = project.hasProperty('wordpress-host') ? project.property('wordpress-host') : 'localhost'
+    scheme = 'https'
+}
+
+def stage = project.findProperty('stage')
```

Comment:
how is that defined externally?

Reply:
I've added documentation, it's a project property: https://docs.gradle.org/current/userguide/build_environment.html#sec:project_properties